build.gradle

```diff
@@ -67,32 +67,32 @@ sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
 
 intellij {
-    version ideVersion
-    pluginName props.getProperty('plugin.name').toLowerCase().replace(' ', '-')
-    downloadSources Boolean.valueOf(downloadIdeaSources)
+    version = ideVersion
+    pluginName = props.getProperty('plugin.name').toLowerCase().replace(' ', '-')
+    downloadSources = Boolean.valueOf(downloadIdeaSources)
     updateSinceUntilBuild = true
     if (hasPycharmPath) {
-        alternativeIdePath pycharmPath
+        alternativeIdePath.set(pycharmPath)
```

Comment:
Hi @intgr, just curious: why using `.set()` here instead of equal like in the other lines?

Reply:
Ah this is just an oversight, I was initially confused about the differences between Groovy and Kotlin DSL.